### PR TITLE
Complete Scripts Description

### PR DESCRIPTION
**Pull request by Xunzhuo**
Add 4 new Scripts to README

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -88,3 +92,70 @@ Options:
...
+  + /docs/bar                : is a redirect entry, or
+  + /docs/bar                : is something we don't understand
+
+```
+## lsync.sh
+
+This script checks if the English version of some localized contents have changed 
+since a localized version has been committed.
+
+The following example check a single file:
```
Comment (sftim) [inline scripts/README.md:147]: nit:
```suggestion
The following example checks a single file:
```

File: scripts/README.md [hunk 2]
```diff
@@ -88,3 +92,70 @@ Options:
...
+
+      ./scripts/lsync.sh content/zh/docs/concepts/
+
+## replace-capture.sh
+
+This script sets K8S_WEBSITE in your env to your docs website root or rely on this script to determine it automatically
+
+You must run the script inside the repository for that to work
+
+    $ ./scripts/replace-capture.sh
```
Comment (sftim) [inline scripts/README.md:161]: BTW this tool is obsolete, we could actually remove it.

Comment (sftim) [reply scripts/README.md:161]: It was used for the migration from Docsy. It's a one-time conversion tool.

**Review by sftim — COMMENTED** [review]
Looks OK to me; one of the descriptions is a bit misleading.

File: scripts/README.md (hunk 2 above)
Comment (Xunzhuo) [reply scripts/README.md]: OK, done. Please take a look.

**Pull request merged**